examples/e-commerce/src/widgets/Pagination.js

```diff
@@ -3,7 +3,7 @@ import { pagination as paginationWidget } from 'instantsearch.js/es/widgets';
 const pagination = paginationWidget({
   container: '#pagination',
   scrollTo: '.container',
-  totalPages: 4,
+  totalPages: 5,
```

Comment:
This option limit the number of pages to 5, this is really what we want?

![Screenshot 2019-06-05 at 14 44 51](https://user-images.githubusercontent.com/6513513/58957239-82c55b80-87a0-11e9-8e4d-ebafe73f9cc2.png)

Reply:
From our discussion with Nico, there shouldn't be a reason to go beyond because the first pages should already be relevant to the query. Do you think using `padding` would be a better alternative?

Reply:
Ok, it doesn't matter much at the end it's an example. On real website though (it depends on your strategy) I would have used padding. Users might want to be able to browse the full content of the catalog e.g. Amazon, Google, leboncoin.